### PR TITLE
Create Min num of operations to make array equal .

### PR DESCRIPTION
**Pull request by SushanthReddyManda**
Fixes issue  #83

### CODE REVIEW

**Comment by SushanthReddyManda** [comment]
Closes #83

**Comment by SushanthReddyManda** [comment]
Closes #83

File: Python/Minimum Operations to Make Array Equal.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+## Code
+
+```
+class Solution:
+    def minOperations(self, n: int) -> int:
+        if n%2 ==0:
+            return int((n/2)**2)
+        else:
+            return int(((n-1)*(n+1)/4))
```
Comment (noisefilter19) [inline Python/Minimum Operations to Make Array Equal.py:30]: It would be helpful to add how we arrive at this formula

**Review by noisefilter19 — CHANGES_REQUESTED** [review]
Please make the requested change

File: Python/Minimum Operations to Make Array Equal.py (hunk 1 above)
Comment (SushanthReddyManda) [reply Python/Minimum Operations to Make Array Equal.py:30]: Ok i will add it

**Comment by SushanthReddyManda** [comment]
@noisefilter19  Added Explanation to above code

**Review by noisefilter19 — APPROVED** [review]

**Pull request merged**